spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -110,7 +109,7 @@ private void runExec() {
 			logger.info("Starting tail process");
 		}
 		try {
-			Process process = Runtime.getRuntime().exec(this.command);
+			Process process = Runtime.getRuntime().exec(getCommand());
```

Comment:
I called that `command "compilation"` not for nothing :smile: . There would really some impact (minor of course), that we build the `command` var on each call for `runExec()`. No?